Pipfile

```diff
@@ -11,12 +11,15 @@ isort = "~=5.10.0"
 mypy = "~=0.950"
 pylint = "~=2.13.0"
 decorator = "~=5.1"
+pytest = "~=7.1.2"
+panther_core = "*"
```

Comment:
cc @panther-labs/detections 

not sure what the current stance on PAT vs PAC is

Reply:
This should be sorted out before merging.

Reply:
@nmartin413 Can you please chime in here?